static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -511,24 +595,66 @@ const SubText = styled('div')`
 `;
 
 const DisplayNameLink = styled(Link)`
-  font-size: ${p => p.theme.fontSize.lg};
+  color: ${p => p.theme.textColor};
+  font-size: ${p => p.theme.fontSize.md};
+  font-weight: ${p => p.theme.fontWeight.bold};
   line-height: normal;
   ${p => p.theme.overflowEllipsis};
 
-  font-weight: ${p => p.theme.fontWeight.bold};
-  &[data-has-viewed='true'] {
-    font-weight: ${p => p.theme.fontWeight.normal};
```

Comment:
has-viewed state is in the new column, a little circle indicator thing with tooltip.